include/boost/math/statistics/bivariate_statistics.hpp

```diff
@@ -17,11 +17,10 @@
 #include <algorithm>
 #include <cmath>
 #include <cstddef>
-#include <boost/assert.hpp>
 
 // Support compilers with P0024R2 implemented without linking TBB
 // https://en.cppreference.com/w/cpp/compiler_support
-#ifndef BOOST_NO_CXX17_HDR_EXECUTION
+#if !defined(BOOST_NO_CXX17_HDR_EXECUTION) && (__cplusplus > 201700L || _MSVC_LANG > 201700L)
```

Comment:
This is Boost.Config's responsibility to get right, plus this should already be handled here: https://github.com/boostorg/config/blob/1c45d765f107e0fcfc45a075c55bd66780e98188/include/boost/config/detail/suffix.hpp#L1096 

What am I missing?

Reply:
That looks correct. Is that new to the develop branch? I link against release 1.75 and it only has optional, string_view, and variant.

Reply:
Yes develop.